packages/node/src/sdk.ts

```diff
@@ -111,6 +124,32 @@ export function init(options: NodeOptions = {}): void {
   initAndBind(NodeClient, options);
 }
 
+/**
+ * Auto-discover integrations based on package.json entries
+ */
+function discoverIntegrations(): Integration[] {
+  const pkg = readPkgUp();
+
+  if (!pkg) {
+    return [];
+  }
+
+  return Object.keys({
+    ...pkg.packageJson.dependencies,
+    ...pkg.packageJson.devDependencies,
+  })
+    .filter(name => {
+      return /^@sentry\/integration-(common|node)-[a-z]/.test(name);
```

Comment:
Missing something in `/^@sentry\/integration-(common|node)-[a-z]/`?

Reply:
We are only concerned about the package prefix, so this regexp is valid.